src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,84 @@
+package duke;
+
+import java.util.Scanner;
+
+import duke.task.TaskList;
+
+
+
+/**
+ * UI class that deals with interactions with the user.
+ */
+public class Ui {
+
+    /**
+     * Method that simply scan the user's input and change it to a command.
+     *
+     * @return a user command as String
+     */
+    public String readCommand() {
+        Scanner sc = new Scanner(System.in);
+        String str = sc.nextLine();
+        return str;
+    }
+
+    /**
+     * Method that load and print the task list.
```

Comment:
instead of print the task list, this method actually just return the printing message.